.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: ruby
+
+rvm:
+  - 3.0
+  - 3.1
+
+sudo: false
+
+bundler_args: --without production
+
+script:
+  - bundle exec rspec spec/
```

Comment:
I can anticipate a question: "What about rubocop?" :stuck_out_tongue_winking_eye: 

First _(besides my well-known feelings toward this... creature :rage:)_ is that adding it now would force us to change hundreds (or more! :scream: !!) files with absolutely irrelevant changes, and then nobody would be able to understand what changes were actually necessary for the autoloading stuff.

Second... there is no second reason, I think. :thinking: